src/components/Navbar/Cart.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '../../App.css';
+import { CartItems } from './CartItems';
+import cart from './icons/cart.svg';
+
+class Cart extends React.Component {
+    state = { dropdownVisible: false } 
+
+    handleDropdownClick = () => {
+        this.setState(({ dropdownVisible }) => ({
+            dropdownVisible: !dropdownVisible,
+        }))
+    }
+
+    render() {
+        const dropdownVisible = this.state.dropdownVisible;
```

Comment:
i think the variable is not needed just use this.state.dropdownVisible

Reply:
deleted